### PR TITLE
DEVDOCS-4719: [migrate] Building Channels, migrate syntax

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4719]

Migration syntax for Building Channels section

[DEVDOCS-4719]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bc-tgomez — APPROVED** [review]
Made a suggestion you can take or leave. Looks good!

File: docs/api-docs/channels/channel-app-best-practices.mdx [hunk 1]
```diff
@@ -1,89 +1,89 @@
...
-- [BigDesign Component Library](https://developer.bigcommerce.com/big-design/?path=/story/badge--overview)
-- [Sell Everywhere with Channel Manager](https://support.bigcommerce.com/s/article/Selling-Everywhere-with-Channel-Manager)
-
-### Tools
-
-- [Channels Sample App](https://github.com/bigcommerce/channels-app)
+# Channel App Best Practices
+
+<!-- Dev Center URL: https://developer.bigcommerce.com/api-docs/channels/guide/channel-app-best-practices -->
+
```
[the remark this answers is not in the document]
Comment (bc-andreadao) [reply docs/api-docs/channels/channel-app-best-practices.mdx:4]: ✔️

File: docs/api-docs/channels/channel-app-requirements.mdx [hunk 2]
```diff
@@ -1,68 +1,66 @@
...
+
+- Includes `config_meta.app.id` in the [create channel](/api-reference/store-management/channels/channels/createchannel) request.
+
+<Callout type="info">
+  - You can find the app ID in the URL when editing the app in the [Developer Portal](https://devtools.bigcommerce.com/). For more information, see [Finding an App's ID](/api-docs/apps/tutorials/id).
+  - `config_meta.app.id` is optional; however, if you're building an app that creates or manages a channel, we recommend including the app ID to ensure the user interface in the BigCommerce control panel works properly.
+  - Select partners who are promoted in the Channel Manager must build an app, and include the app ID in the create channel request.
+</Callout>
+
+
```
[the remark this answers is not in the document]
Comment (bc-andreadao) [reply docs/api-docs/channels/channel-app-requirements.mdx]: ✔️

**Review by slsriehl — APPROVED** [review]
aprrrove

**Pull request merged**